jax/interpreters/xla.py

```diff
@@ -134,7 +134,7 @@ def xla_primitive_callable(prim, *abstract_args, **params):
     handle_result = aval_to_result_handler(aval_out)
   xla_shapes = tuple(map(aval_to_xla_shape, abstract_args))
   built_c = primitive_computation(prim, *xla_shapes, **params)
-  compiled = built_c.Compile(xla_shapes, xb.get_compile_options(),
+  compiled = built_c.Compile(None, xb.get_compile_options(),
```

Comment:
Note these arguments are kwargs (https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/xla/python/xla_client.py#L555) so you  could probably just write `Compile(compile_options=...)` rather than adding a `None` argument.

Same everywhere.

Reply:
Done.